.gitignore

```diff
@@ -3,3 +3,4 @@
 dist-newstyle/
 dist/
 cabal.project.local
+.vscode
```

Comment:
tip: `echo .vscode >> .git/info/exclude` to have it locally ignored, though I don't mind adding it.